rustworkx/rustworkx.pyi

```diff
@@ -133,6 +133,18 @@ def graph_closeness_centrality(
     graph: PyGraph[_S, _T],
     wf_improved: bool = ...,
 ) -> CentralityMapping: ...
+def digraph_newman_weighted_closeness_centrality(
+    graph: PyDiGraph[_S, _T],
+    weight_fn: Callable[[_T], float],
```

Comment:
Because of `default_weight`, we could in theory accept `None` as an argument for this
```suggestion
     weight_fn: Callable[[_T], float] | None = ...,
```